agent-tooling/src/main/java/io/opentelemetry/auto/tooling/TracerInstaller.java

```diff
@@ -70,10 +70,12 @@ private static synchronized void installExportersFromJar(final String exporterJa
         getExporterFactory(SpanExporterFactory.class, exporterLoader);
     if (spanExporterFactory != null) {
       final SpanExporter spanExporter = spanExporterFactory.fromConfig(config);
-      OpenTelemetrySdk.getTracerProvider()
-          .addSpanProcessor(
-              BatchSpansProcessor.create(
-                  spanExporter, BatchSpansProcessor.Config.loadFromDefaultSources()));
+      BatchSpanProcessor spanProcessor =
+          BatchSpanProcessor.newBuilder(spanExporter)
+              .readEnvironmentVariables()
+              .readSystemProperties()
```

Comment:
System properties override environment variables 👍

This seems good to me, same as Spring Boot: https://docs.spring.io/spring-boot/docs/current/reference/html/spring-boot-features.html#boot-features-external-config

Reply:
Yes :) I always use their order as a reference :)